ocaml/tests/test_vm_helpers.ml

```diff
@@ -524,6 +528,38 @@ let test_group_hosts_netsriov_with_allocated () =
             "Test-failure: Unexpected number of sriov network in test"
   )
 
+(*--- Helper functions ---*)
+let on_pool_of_anti_affinity
```

Comment:
I can't see anti-affinity group is created or configured for VM in the func?

Reply:
AA group is created and configured in the outer function. Not here. 
I have removed this function for it just creates 3 hosts without other purposes.